VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -1,113 +1,74 @@
+        isRequiredField={false}
+        testID={'InputTest'}
+        isTextArea={false}
+        error={''}
+      />)
     })
-  })
-
-  describe('when input type is email', () => {
-    it('should set the textContentType to "emailAddress"', async () => {
-      expect(testInstance.findByType(TextInput).props.textContentType).toEqual('emailAddress')
-      expect(testInstance.findByType(TextInput).props.keyboardType).toEqual('email-address')
+    it('should call onChange when text is entered', () => {
+      fireEvent.changeText(screen.getByTestId('InputTest'), 'content')
+      expect(onChangeSpy).toHaveBeenCalled()
     })
-  })
 
-  describe('when input type is phone', () => {
-    it('should set the textContentType to "telephoneNumber"', async () => {
-      initializeTestInstance('phone')
-      expect(testInstance.findByType(TextInput).props.textContentType).toEqual('telephoneNumber')
+    it('should display a label if provided', () => {
+      expect(screen.getByText('Email address')).toBeTruthy()
     })
-  })
 
-  describe('when there is helper text', () => {
-    it('should display it', async () => {
-      initializeTestInstance('email', '', 'back.a11yHint')
+    it('should display helper text if provided', () => {
       expect(screen.getByText('Navigates to the previous page')).toBeTruthy()
-      expect(testInstance.findByProps({ 'children': 'Navigates to the previous page' })).toBeTruthy()
     })
   })
 
   describe('when there is an error', () => {
-    it('should display it', async () => {
-      initializeTestInstance('email', '', '', 'ERROR')
+    beforeEach(() => {
+      render(<VATextInput
+        inputType={'email'}
+        onChange={onChangeSpy}
+        labelKey={'contactInformation.emailAddress'}
+        value={''}
+        helperTextKey={'back.a11yHint'}
+        isRequiredField={false}
+        testID={'InputTest'}
+        isTextArea={false}
+        error={'ERROR'}
+      />)
+    })
+    it('should display Error text', () => {
       expect(screen.getByText('ERROR')).toBeTruthy()
-      expect(testInstance.findByProps({ 'children': 'ERROR' })).toBeTruthy()
     })
+  })
 
-    it('should set the border color to error and make the border thicker', async () => {
-      initializeTestInstance('email', '', '', 'ERROR')
-      expect(testInstance.findByProps({ 'borderColor': 'error' })).toBeTruthy()
-      expect(testInstance.findByProps({ 'borderWidth': 2 })).toBeTruthy()
+  describe('when it is a required field', () => {
+    beforeEach(() => {
+      render(<VATextInput
+        inputType={'email'}
+        onChange={onChangeSpy}
+        labelKey={'contactInformation.emailAddress'}
+        value={''}
+        helperTextKey={'back.a11yHint'}
+        isRequiredField={true}
+        testID={'InputTest'}
+        isTextArea={false}
+        error={''}
+      />)
+    })
+    it('should include "(Required)" in the label', () => {
+      expect(screen.getByText('(Required)', { exact: false })).toBeTruthy()
```

Comment:
This can be slightly simplified:

```
expect(screen.getByText(/(Required)/)).toBeTruthy()
```

Reply:
Updated here and other places it was testing for `(Required)` text.